modules/local/mtx_to_h5ad.nf

```diff
@@ -28,6 +28,8 @@ process MTX_TO_H5AD {
     script:
     def aligner = (input_aligner in [ 'cellranger', 'cellrangerarc', 'cellrangermulti' ]) ? 'cellranger' : input_aligner
 
+    aligner = input_aligner == "alevin" ? "simpleaf" : aligner
```

Comment:
Is this for backwards compatibility? 
Can't we fix this at an earlier point in the pipeline, i.e. at the very beginning set aligner to `simpleaf` if it's `alevin` and then use `simpleaf` throughout the pipeline? 

I'm afraid one needs to fix it at multiple locations otherwise.

Reply:
> This will avoid all confusions, but will change the file structure and the default "aligner" option. What do you think?

I'm all for changing this! For backwards compatibility, we can then simply add an alias for "alevin" at the beginning when evaluating the parameters.